pkg/api/v1beta3/dynakube/telemetryservice/props.go

```diff
@@ -3,6 +3,8 @@ package telemetryservice
 type Protocol string
 
 const (
+	telemetryServiceSuffix = "-telemetry"
```

Comment:
you will hate me: 😭 

name of package `telemetryservice`
name of const `telemetryServiceSuffix`

could we improve that? 😅 
- `nameSuffix`
- `suffix`

Reply:
I didn't want `suffix` so I kept `telemetryServiceSuffix`

I like `nameSuffix`

Reply:
That's a good point; there's there is no need to duplicate the package's name inside variables or constants.